homework-04/middlewares/cookieParser.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express'
+import cookieParser from 'cookie-parser'
+
+const router = express.Router()
+router.use(cookieParser())
+
+router.use((req, res, next) => {
```

Comment:
this middleware doesn't make sense: according to task you should have created your own cookie parser middleware, but here you're using cookies from `cookie-parser`

Reply:
fixed